bluebird-promisell.js

```diff
@@ -0,0 +1,47 @@
+/* jshint esnext:true */
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+const Promise = require('bluebird');
+const P = require('bluebird-promisell');
+const R = require('ramda');
+const S = require('sanctuary');
+
+//    readFile :: Object -> String -> Promise Error String
+const readFile = S.flip(Promise.promisify(fs.readFile));
```

Comment:
I'm keen to avoid unnecessary differences between the implementations. `readFile` in [tasks.js][1] takes `(encoding, filename)`. Let's do the same here:

```javascript
//    data Text = Buffer | String
//    readFile :: String -> String -> Promise Error Text
const readFile =
R.curry((encoding, filename) => fs.readFile(filename, {encoding: encoding}));
```


[1]: https://github.com/plaid/async-problem/blob/master/tasks.js